src/main/java/leave_request/LeaveRequestDsGateway.java

```diff
@@ -0,0 +1,5 @@
+package leave_request;
+
+public interface LeaveRequestDsGateway {
+    void save(LeaveRequestDsRequestModel requestModel);
```

Comment:
Are you talking about if someone makes another leave request when they already made one?

Reply:
Not really. What I mean is that our entity will not always exists and will need to be reload every time we run the program. So if I, as an employee, requested a leaving request yesterday, and I log in to the system again today, I will need to load the leavingRequest Project from our database. So load method should be necessary.

Reply:
But since LeaveRequest will just be a Project that the user is the head of, wouldn't this be the same for every other Project the user is a part of and thus I don't need to do something specific here?